newsfragments/2772.feature.rst

```diff
@@ -1 +1 @@
-Update WorkerPool error messages returned by Porter API.
\ No newline at end of file
+Update WorkerPool error messages returned by Porter API.
```

Comment:
Huh? What changed here?

Reply:
A newline was missing - failed one of the pre-commit checks - https://github.com/nucypher/nucypher/pull/2768/commits/ba0c2bd056930e99d139715af0917a2127b5edfa

Reply:
Someone is not using git hooks? 🤔